tracer/src/Datadog.AutoInstrumentation.NativeLoader/instrumented_assembly_generator/instrumented_assembly_generator_cor_profiler_function_control.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <atomic>
+#include <corhlpr.h>
+#include <corprof.h>
+
+namespace instrumented_assembly_generator
+{
+class InstrumentedAssemblyGeneratorCorProfilerFunctionControl : public ICorProfilerFunctionControl
+{
+private:
+    std::atomic<int> m_refCount;
+    ICorProfilerFunctionControl* m_pICorProfilerFunctionControl;
+    ICorProfilerInfo12* m_corProfilerInfo;
+    ModuleID m_moduleId;
+    mdMethodDef m_methodId;
+
+public:
+    explicit InstrumentedAssemblyGeneratorCorProfilerFunctionControl(ICorProfilerFunctionControl* corProfilerFunctionControl,
```

Comment:
explicit is not needed here:
explicit is needed if your constructor has only one parameter.

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a